### PR TITLE
Relase Tensorflow-Lite inference from Tensorflow

### PR DESCRIPTION
**Pull request by MrRiahi**
In this pull request, I share my inference code for pose estimation only using Tensorflow-Lite, OpenCV, Pillow, and numpy packages.

### CODE REVIEW

File: bodypix_tflite/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# BodyPix Tensorflow-lite Inference
+First install tensorflow-lite.
+
+```shell
+pip install --extra-index-url https://google-coral.github.io/py-repo/ tflite_runtime
```
Comment (de-code) [inline bodypix_tflite/README.md]: Does this mean you are actually using the Coral TPU?
Or can that version of TensorFlow Lite also be used without the Coral TPU?

**Comment by de-code** [comment]
Thank you for creating the PR.

I can see there is now the [tflite-runtime package on PyPI](https://pypi.org/project/tflite-runtime/).
Does that work for you as well?

There are a few things that I think we need to do:
- Merge `bodypix_tflite/tf_bodypix` with `tf_bodypix` (otherwise it's like a separate project):
  - When merging, the current functionality should still work, imports may need to be dynamic
- Remove `bodypix_tflite` (with it the models that should be downloaded on demand, but may need to host he tflite version)
- Add a ci test to test with `tflite-runtime` rather than `tensorflow` dependency
- The `tflite-runtime` dependency could be expressed as a Python extra

If you happy, then I could make some of those changes in your branch.

File: bodypix_tflite/README.md (hunk 1 above)
Comment (MrRiahi) [reply bodypix_tflite/README.md]: Hi

I check in TensorFlow website to install tflite package in python and found this command. This Tensorflow-Lite can use without Coral TPU.

**Comment by MrRiahi** [comment]
I did not check the code with tflite-runtime package on PyPI.

If I should do anything, please tell me. Thanks.

**Comment by de-code** [comment]
> If I should do anything, please tell me. Thanks.

The points mentioned my previous comment (https://github.com/de-code/python-tf-bodypix/pull/157#issuecomment-995302625) will need to be done. Basically the goal is that this library (installed via `pip`) works out of the box and transparently with `tflite-runtime` (rather than `tensorflow`).

I could make a start making those changes in your PR branch. Are you happy for me to make those changes in your PR branch?

**Comment by MrRiahi** [comment]
Yes, I would be happy to make those changes in my PR branch.

**Comment by de-code** [comment]
> Yes, I would be happy to make those changes in my PR branch.

Great, thank you. Can you give me permissions to your PR branch please?

See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

(Although I am not sure whether the option is there because you have using the `develop` branch. Potentially you'd need to create a separate branch and switch the PR to the new branch)

**Comment by MrRiahi** [comment]
Sorry. This is the first contribution I have in a project and therefore I am not familiar with that. If it is applicable, I or you can finish this PR and create a new one in another branch.
Also, I check and find out that the `Allow edits by maintainers` is ticked.

**Comment by de-code** [comment]
> Sorry. I do not contribute to any project until now and therefore I am not familiar with that. If it is applicable, I or you can finish this PR and create a new one in another branch.

That is fine. There is always a first time :-)
It is good to contribute open source projects.
I should add a contributing guide.

Generally it is advised to create a separate branch in your fork and create the PR from that new branch.
But in the end it doesn't affect the outcome as much. However, in order for me to contribute to the PR, I will need access.
Usually there is a tick box when creating a PR. I am not sure if it is shown for you in this case, because the PR branch would be the main branch of your fork.

If you don't see that option, you could try to create a branch from the `develop` branch of your fork (which will already have your changes in it), then you can edit this PR and change the from branch to the new branch. That will then hopefully show the option to allow edits to the PR branch by the project maintainers.

Alternatively you could also add me to your fork with "Write" permissions. But going through the above approach would be preferable as that it how it would typically work for your future contributions to other open source projects.

**Pull request closed without merging**